jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/AbstractRuntimeManager.java

```diff
@@ -169,12 +171,18 @@ protected void registerItems(RuntimeEngine runtime) {
         for (AgendaEventListener listener : agendaListeners) {
             ksession.addEventListener(listener);
         }
-        
+
+        if (factory instanceof SimpleRegisterableItemsFactory) {
+            List<NotificationListener> additionalNotificationListener = ((SimpleRegisterableItemsFactory) factory).getNotificationListeners(runtime);
+            NotificationListenerManager.get().registerAdditionalNotificationListener(additionalNotificationListener);
```

Comment:
@elguardian wouldn't this be enough to make the changes in kie-spring-boot modules and just call `NotificationListenerManager.get().registerAdditionalNotificationListener` from there?